canonical-web-and-design/ubuntu.com
# Joe Sandoval masters page

## Done

- Added Joe Sandoval masters conference page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/masters-conference
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page matches the [copy doc](https://docs.google.com/document/d/1Z63eMCe9o4Onk2ezPFLPDTbI-uJezAOzI-aSpXmZ5lc/edit)
- Find the "Read more about managing a large and diverse infrastructure fleet" link, click it
- See that the page matches the [copy doc](https://docs.google.com/document/d/1Z63eMCe9o4Onk2ezPFLPDTbI-uJezAOzI-aSpXmZ5lc/edit)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2190
